modules/edu.gemini.giapi.client/src/main/scala/edu/gemini/giapi/client/gpi/GPIClient.scala

```diff
@@ -153,18 +161,20 @@ object GPIExample extends App {
 
   import cats.effect.IO
   import scala.concurrent.duration._
+  import cats.instances.string._
 
   private val gpiStatus =
     Stream.bracket(
       Giapi.giapiConnection[IO]("failover:(tcp://127.0.0.1:61616)", 2000.millis).connect)(
       giapi => {
-        val client = new GPIClient[IO](giapi)
+        val client = new GPIClient[IO](giapi, scala.concurrent.ExecutionContext.Implicits.global)
```

Comment:
Shouldn't this be `ec` ?

Reply:
There is no ec in this context. you need to pass an `ExecutionContext` and I'm just passing `global` directly